Updates routing logic

TimRobinson1 opened the pull request:
Quick update to the routing logic - if the user is currently signed in, root takes you to THE WALL.  Else, it takes you to the signup page.

neoeno commented on spec/controllers/home_controller_spec.rb:
```diff
@@ -0,0 +1,5 @@
+
+RSpec.describe HomeController, type: :controller do
+
+end
```
I'd suggest deleting this file

neoeno commented on app/assets/stylesheets/home.scss:
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Home controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
I'd suggest deleting this file

neoeno commented on app/assets/javascripts/home.js:
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
I'd suggest deleting this file

TimRobinson1 commented:
We added some changes, including using the built in Clearance constraints (thank you, Kay!) and removed the un-needed files.  Users who are not signed in now have no access to posts and must first sign_up/sign_in

neoeno commented:
Great! I'd delete home_controller, home_helper, and home_controller_spec. I think I'm right in saying they're no longer required?

TimRobinson1 commented:
Right, I've removed those, you're right about them being no longer required.

neoeno commented on app/helpers/home_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```
One more!

neoeno commented:
Great work!

neoeno commented:
(Go ahead and merge when you're ready, btw!)
